app/code/Magento/SalesGraphQl/etc/schema.graphqls

```diff
@@ -199,6 +200,40 @@ type PaymentMethod @doc(description: "Contains details about the payment method
     additional_data: [KeyValue] @doc(description:  "Additional data per payment method type")
 }
 
+type CreditMemo @doc(description: "Credit memo details") {
+    id: ID! @doc(description: "the ID of the credit memo, used for API purposes")
+    number: String! @doc(description: "sequential credit memo number")
+    state: String!
```

Comment:
I believe we missed this in the proposal https://github.com/magento/architecture/blob/master/design-documents/graph-ql/coverage/customer-orders.md 
I would be inclined to name is status rather than state (see ProductStockStatus for example)
State is ambiguous because it's used in addresses as a region.
I'll submit this for an approval

Reply:
We should not expose `state` of order, invoice, credit memo entities to the storefront, `status` should be used instead. The `status` was specially introduced for storefront API.